share/spack/qa/configuration/packages.yaml

```
@@ -12,8 +12,8 @@ packages:
   perl:
     buildable: False
     externals:
-    - spec: perl@5.26.1
-      prefix: /usr
+    - spec: perl@5.32.0
+      prefix: /opt/local
```

```suggestion
    - spec: perl@5.26.1
      prefix: /usr
```
this was added by accident I suppose?

---

ouch, yes, it was accidental